M2/Macaulay2/e/BasicPolyList.cpp

```diff
@@ -40,7 +40,7 @@ void BasicPolyListStreamCollector::appendExponent(VarIndex index, Exponent expon
   mValue[mCurrentPoly].mMonomials[mSizeEntryInMonomial] += 2;
 }
 
-void BasicPolyListStreamCollector::appendTermDone(Coefficient coefficient)
+void BasicPolyListStreamCollector::appendTermDone(const Coefficient& coefficient)
 {
   mValue[mCurrentPoly].mCoefficients[mCurrentTerm] = coefficient;
 }
```

Comment:
Is this correct?  Does this correctly copy the element? I think it probably does, just want to flag that